docs/faq.md

```diff
@@ -21,7 +21,7 @@ Arrow functions **are not** synonymous with normal functions. `arguments` and `t
 reference their *outer function* for example:
 
 ```javascript
-var user = {
+const user = {
```

Comment:
The eslint configuration prefers const. Given that most modern browser support `const`, I think that's fine.